test/unit/org/apache/cassandra/index/sai/postings/IntArrayPostingListTest.java

```diff
@@ -15,25 +15,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.cassandra.index.sai.utils;
+package org.apache.cassandra.index.sai.postings;
 
 import org.junit.Test;
 
 import org.apache.cassandra.index.sai.disk.PostingList;
+import org.apache.cassandra.index.sai.utils.SaiRandomizedTest;
 
-public class ArrayPostingListTest extends SaiRandomizedTest
+public class IntArrayPostingListTest extends SaiRandomizedTest
 {
     @Test
     public void testArrayPostingList() throws Exception
     {
-        ArrayPostingList postingList = new ArrayPostingList(new int[]{ 1, 2, 3 });
+        IntArrayPostingList postingList = new IntArrayPostingList(new int[]{ 1, 2, 3 });
```

Comment:
Mostly unrelated to the changes, but since we are here we could use try-with-resources to create the posting lists, as suggested by the warning. Feel free to ignore it if you prefer to.

Reply:
I look to other similar usages in tests:
- Either the same warning
- Or annotates to ignore the warning
So I believe it's better to fix together with other files in a different PR.

Reply:
Agree, let's do it in a separate PR.